BaseFilter/sections/specific.txt

```diff
@@ -4581,6 +4581,8 @@ wiki.gg##div[class^="sidebar-showcase"]
 wiki.gg##div[class^="games-showcase"]
 pornone.com##body .vjs-paused > .warp
 earth.com##div[class^="PrebidAdSlot_"]
+tutubox.io###vipModal > div.modal-content > iframe.vip-upsell__iframe:last-child
+store.app-valley.vip#$?#.modal1 { remove: true; }
```

Comment:
On my end `tutubox.io,store.app-valley.vip###vipModal` works fine.

Reply:
It does? That's odd

Reply:
You're right!!!, changed it to your not css-based Solution.